src/main/resources/db/changelog/changes/07-add-users-roles-table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+    - changeSet:
+          id: update-users-roles-table
+          author: mpbakunov
+          changes:
+              - sql:
+                    sql: |
+                        INSERT INTO users_roles (user_id, role_id)
+                        SELECT u.id, r.id
+                        FROM users u, roles r
+                        WHERE r.name = 'USER';
```

Comment:
Why do you need it if you give a user role during the registration flow?

Reply:
**This script was necessary in the case when there were users in the User table before the new changes were set.** The goal is to assign all these users the USER role.